apps/app_rpt.c

```diff
@@ -2695,8 +2695,12 @@ static int rpt_setup_channels(struct rpt *myrpt, struct ast_format_cap *cap)
 		myrpt->txchannel = myrpt->rxchannel;
 		myrpt->dahditxchannel = IS_DAHDI_CHAN_NAME(myrpt->rxchanname) && !IS_PSEUDO_NAME(myrpt->rxchanname) ? myrpt->txchannel : NULL;
 	}
+	/* If the TX channel is not a pseudo channel, toggle the PTT line */
 	if (!IS_PSEUDO(myrpt->txchannel)) {
+		/* Wait for the driver to finish initialization */
+		usleep(250000);
```

Comment:
Do not use usleep when sleeping on a channel, use `ast_safe_sleep` instead.
Keep in mind the latter uses ms, not us like usleep.